parlai/core/torch_agent.py

```diff
@@ -941,6 +948,11 @@ def report(self):
             current_lr = round_sigfigs(self.optimizer.param_groups[0]['lr'], 4)
             metrics['lr'] = round_sigfigs(current_lr, 4)
         metrics['num_updates'] = self._number_training_updates
+
+        steps = max(1, self.metrics['updates'])  # prevent divide by zero
```

Comment:
again,  i don't think this is  in the right place. the history objects doesn't have a metrics attribute

Reply:
O_o what in teh world!